AllReadyApp/Web-App/AllReady/Features/Requests/AddRequestCommandHandler.cs

```diff
@@ -42,7 +42,8 @@ public AddRequestCommandHandler(AllReadyContext context, IGeocoder geocoder)
 
             var address = _geocoder.Geocode(message.RequestViewModel.Address, message.RequestViewModel.City, message.RequestViewModel.State, message.RequestViewModel.Zip, string.Empty).FirstOrDefault();
             request.Latitude = message.RequestViewModel.Latitude == 0 ? address?.Coordinates.Latitude ?? 0 : message.RequestViewModel.Latitude;
-            request.Longitude = message.RequestViewModel.Latitude == 0 ? address?.Coordinates.Longitude ?? 0 : message.RequestViewModel.Latitude;

```

Comment:
@stevejgordon, yes please, the whole second line should be dealing with Longitude, not Latitude. Thanks